### DISCUSSION TITLE
Foundry NFT's  can't see the basic nft image uri in metamask when deployed and minted the NFT

### DISCUSSION THREAD

**Discussion by NeoRusi** [Cyfrin/foundry-full-course-cu#1000] [Q&A]
I am trying to use another Image from my own custom file, and I have problems in deploying It to IPFS

1) I added the NFT imageUri here 
<img width="1276" alt="Screenshot 2023-10-12 at 17 03 21" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/103575933/5fcb858e-a6d3-4055-886a-26d2e6f9a40c">
it uploaded succesfully
<img width="1277" alt="Screenshot 2023-10-12 at 17 03 36" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/103575933/a2235682-4b55-4568-81c0-6afce00d7d30">
2) Then I take the CID 
<img width="1440" alt="Screenshot 2023-10-12 at 17 04 42" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/103575933/71b37ece-4171-4887-9986-bbd1806308a9">
3) add it to the ipfs gateway
ipfs://QmV84KcFWhydmhLrXXJDs9kNUCJGappmCAsPJ1P1sNmZXn

4) I replace the given imageUri of the PUG with this imageURI
and I can't see it in metamask after deploying and minting successfully

**Comment by 0xusmanf** [comment]
I guess you are missing creating the json file. Create the json file with the imageURI you created and upload it to IPFS the same way you uploaded your image and replace that json file's URI with that pug URI.

**Comment by theirrationalone** [comment]
Pinning might be missing.....
have you verified that???
;`)
